homework9/src/components/TeacherCard/index.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+import "./TeacherCard.css";
+
+export default function TeacherCard({ teacher }) {
+  return (
+    <li className="teacher">
+      <img src={teacher.photo} alt="" className="teacher-picture" />
+      <div className="teacher-container">
+        <h2 className="teacher-name">
+          {teacher.first_name} {teacher.last_name}
```

Comment:
variables with underscore it is more backend style. On FE we usualy use camelCase

Reply:
for transform data from BE u can use `morphism` library